autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -155,6 +166,12 @@ def get_preset_models_regression(path, problem_type, objective_func, stopping_me
             TabularNeuralNetModel(path=path, name='NeuralNetRegressor', problem_type=problem_type,
                                   objective_func=objective_func, stopping_metric=stopping_metric, hyperparameters=nn_options.copy())
         )
+    if (nn_fastai_options is not None) & __is_fastai_available():
+        from ....contrib.tabular_nn_pytorch.nn_tab_model import NNFastAiTabularModel
+        models.append(
+            NNFastAiTabularModel(path=path, name='FastAiNeuralNetClassifier', problem_type=problem_type,
```

Comment:
Can name be shortened eg. to 'FastAiNNClassifier'?  Otherwise name will become very long when suffixes like stacker-level are appended to the name.

Reply:
Also why is this model named '*Classsifier', shouldn't it be named '*Regressor'?  Only the model for classification should have name ending in 'Classifier'

Reply:
Done